libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -1113,6 +1113,40 @@ class AnswerWithJustification(BaseModel):
         else:
             return llm | output_parser
 
+    @beta()
+    def get_num_tokens_from_messages(self, messages: List[BaseMessage]) -> int:
```

Comment:
thoughts on updating base implementation to take tools and/or kwargs? definitely not an anthropic-specific problem

Reply:
id support